backend/song/tests/test_views.py

```diff
@@ -0,0 +1,451 @@
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from backend.users.models import User
+
+from backend.song.models import Artist
+
+
+class BaseTestSetUp(APITestCase):
+    def setUp(self):
```

Comment:
Bro, I am confused here,

since we have already factories and fixtures why not using them here instead of hardcoding the object creation?